sherpa/astro/xspec/__init__.py

```diff
@@ -8200,5 +8206,23 @@ def __init__(self, name='zbabs'):
         self.redshift = Parameter(name, 'redshift', 0.0, 0.0, 1.0e5, 0.0, 1.0e6)
         XSMultiplicativeModel.__init__(self, name, (self.nH, self.nHeI, self.nHeII, self.redshift))
 
+
+@version_at_least("12.9.1")
+class XSbtapec(XSAdditiveModel):
+    """
+
+    """
+
+    __function__ = "C_btapec"
+
+    def __init__(self, name='btapec'):
+        # These parameter values are made up!
```

Comment:
Ouch, I missed that during my final review. One problem is that we have too many "FIXME" and "TODO" comments in the code, so I can't use them to mark what's missing in my implementation... I'll see if my IDE would allow me to use a custom marker.

Reply:
It's also missing the description in the docstring... @DougBurke should I leave that to you?